backend/src/types/integrationEnums.ts

```diff
@@ -14,5 +15,6 @@ export enum IntegrationType {
   GITHUB = 'github',
   TWITTER = 'twitter',
   TWITTER_REACH = 'twitter-reach',
+  HACKER_NEWS = 'hacker-news',
```

Comment:
nit: Let's use either `HACKERNEWS` or `HACKER_NEWS` as enums for `PlatformType` and `IntegrationType` for consistency

Reply:
This was a bug actually :D caught it this afternoon